test/failure_test.py

```diff
@@ -10,14 +10,15 @@
 import time
 import unittest
 
+import ray.ray_constants as ray_constants
 import ray.test.test_functions as test_functions
 
 if sys.version_info >= (3, 0):
     from importlib import reload
 
 
 def relevant_errors(error_type):
-    return [info for info in ray.error_info() if info[b"type"] == error_type]
```

Comment:
does the `b` no longer matter?

Reply:
Due to https://github.com/ray-project/ray/pull/2179/files#diff-8165ed4a37603e0f7e2eaf79e3a69bf7R1166, the `error_info` information consists of string instead of byte objects.